resource-aggregate/pb/commands.proto

```diff
@@ -461,6 +465,7 @@ message UpdateDeviceMetadataRequest {
         ConnectionStatus status = 3;
         ShadowSynchronization shadow_synchronization = 4;
     };
+    int64 valid_until = 99;  // unix timestamp in nanoseconds (https://golang.org/pkg/time/#Time.UnixNano) when command is considered as expired. 0 means forever.
```

Comment:
Everywhere else you incremented by one, here you jumped to 99. Please fix.

Reply:
From my point of view, we need to hold the same number for the same type among all events.

After that user can define his own message for parse only sub-values.